StyleBuilder.js

```diff
@@ -130,7 +131,13 @@ class StyleBuilder {
       [`border${side}Color`]: "initial",
     };
 
-    const values = value.split(" ");
+    const splitValues = value.split(" ");
+    const color = splitValues[2];
+
+    const values = color && _colorValuesThatCouldHaveSpaces.includes(color.split("(")[0])
+      ? [...splitValues.slice(0, 2), splitValues.slice(2, Infinity).join("")]
+      : splitValues
```

Comment:
part of the goal here was to support "1px solid green" and "1px green solid" because currently browsers support all orderings.

What do you think about removing all spaces between `(` and `)` instead?

```
const parenRegex = /\(([^)]+)\)/;
const matches = value.match(parenRegex);
// remove all spaces within color values rgb(...), rgba(...), hsl(...), and hsla(...)
const trimmedValue = matches !== null ? value.replace(matches[0], matches[0].replace(" ", "")).trim() : value.trim();
const values = trimmedValue.split(" ");
```

not sure the performance of regex -- might be worth writing that without regex like below (though jsperf is down)

```
const trimmedValue = function() {
  value = value.trim();
  if (value.indexOf("(") === -1) { return value; }

  const splitOnOpen = value.split("(");
  const splitOnClose = splitOnOpen[1].split(")");
  // this line can be one of two, depends again on performance (though w/ such small strings probably not a big deal
  // I'm guessing the second one below w/ regex is the faster
  return splitOnOpen[0] + "(" + splitOnClose[0].split(" ").join("") + ")" + splitOnClose[1];
  return splitOnOpen[0] + "(" + splitOnClose[0].replace(/ /g, "") + ")" + splitOnClose[1];
}();
```